client/clientservice/src/main.cpp

```diff
@@ -72,6 +78,7 @@ int main(int argc, char** argv) {
   auto metrics = std::make_shared<concordMetrics::Aggregator>();
   concord_client->setMetricsAggregator(metrics);
   ClientService service(std::move(concord_client));
+  LOG_CONFIGURE_AND_WATCH(getLog4CplusConfigLocation(), kLogConfigRefreshIntervalInMs);
```

Comment:
We are retrieving the logger in line 62 and we configure log4cplus in line 81. Shouldn't we configure it before we get the logger or does it not matter? In any case, those two lines should be close together.

Reply:
I checked - it works either way. I moved it to be before line 62.